lms/djangoapps/learner_dashboard/views.py

```diff
@@ -73,4 +74,7 @@ def program_details(request, program_uuid):
         'user_preferences': get_user_preferences(request.user)
     }
 
-    return render_to_response('learner_dashboard/program_details.html', context)
+    if waffle.switch_is_active('new_program_progress'):
+        return render_to_response('learner_dashboard/new_program_details.html', context)
```

Comment:
do you think it is necessary to rename all the existing files that we want to delete? I can do that but it will make all the diffs bigger